### PR TITLE
OSDOCS-34559: updates procedure with xref to alternative option

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OCPBUGS-34559

Link to docs preview:
https://78136--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html#nw-ovn-kubernetes-migration_migrate-from-openshift-sdn

QE review:
- [x] QE has approved this change.

Additional information:
@pliurh PTAL

### CODE REVIEW

File: modules/nw-ovn-kubernetes-migration.adoc [hunk 1]
```diff
@@ -289,6 +289,11 @@ daemon set "multus" successfully rolled out
 
 . To complete changing the network plugin, reboot each node in your cluster. You can reboot the nodes in your cluster with either of the following approaches:
 
+[IMPORTANT]
+====
+The following scripts reboot all of the nodes in the cluster at the same time. This can cause your cluster to be unstable. Another option is to follow the link::https://docs.openshift.com/container-platform/4.15/nodes/nodes/nodes-nodes-rebooting.html#nodes-nodes-rebooting-gracefully_nodes-nodes-rebooting[rebooting a node gracefully] procedure. The procedure reboots the nodes one-by-one so clusters with many nodes will experience considerable downtime.
```
Comment (pliurh) [inline modules/nw-ovn-kubernetes-migration.adoc:294]: Draining node cannot be done when the cluster network is not ready. So a graceful node rebooting is not possible in this case. Users can reboot nodes manually, without cordoning and draining nodes.

Comment (JoeAldinger) [reply modules/nw-ovn-kubernetes-migration.adoc:294]: Would it be helpful to link to this content: https://docs.openshift.com/container-platform/4.16/nodes/nodes/nodes-nodes-rebooting.html? @pliurh

Comment (pliurh) [reply modules/nw-ovn-kubernetes-migration.adoc:294]: I don't think it's helpful here. That link is about how to reboot a node gracefully. However, it does not apply in this situation.

Comment (JoeAldinger) [reply modules/nw-ovn-kubernetes-migration.adoc:294]: The link takes you to the section in the docs that are generally about "rebooting a node".

File: modules/nw-ovn-kubernetes-migration.adoc [hunk 2]
```diff
@@ -289,6 +289,11 @@ daemon set "multus" successfully rolled out
 
 . To complete changing the network plugin, reboot each node in your cluster. You can reboot the nodes in your cluster with either of the following approaches:
 
+[IMPORTANT]
```
Comment (mburke5678) [inline modules/nw-ovn-kubernetes-migration.adoc:292]: I think you need a `+ here. The spacing is off in the preview
```suggestion
+
[IMPORTANT]
```
image

File: modules/nw-ovn-kubernetes-migration.adoc [hunk 3]
```diff
@@ -289,6 +289,11 @@ daemon set "multus" successfully rolled out
...
+====
+The following scripts reboot all of the nodes in the cluster at the same time. This can cause your cluster to be unstable. Another option is to reboot your nodes manually one at a time. Rebooting nodes one-by-one causes considerable downtime in a cluster with many nodes.
+====
```
Comment (mburke5678) [inline modules/nw-ovn-kubernetes-migration.adoc:295]: Does the user know how to reboot nodes manually?

**Comment by pliurh** [comment]
LGTM

File: modules/nw-ovn-kubernetes-migration.adoc (hunk 2 above)
Comment (JoeAldinger) [reply modules/nw-ovn-kubernetes-migration.adoc:292]: Yikes, good catch, thanks.

File: modules/nw-ovn-kubernetes-migration.adoc (hunk 3 above)
Comment (JoeAldinger) [reply modules/nw-ovn-kubernetes-migration.adoc:296]: I think so. The issue here is that we can't reboot gracefully. I checked with QE and another admin I _think_ this is a common enough task that we are ok.

**Pull request merged**